aces/Loris
# [MRI Violations] MRI protocol table is now read-only.

## Brief summary of changes

This PR makes the `mri_protocol` table read-only in the MRI violations module.

#### Testing instructions (if applicable)

Access the MRI violations module and search for problems of type 'Could not identify scan type'. Verify that the `mri_protocol` table on the new page displayed is read-only.

#### Link(s) to related issue(s)

* Resolves #6310
